contracts/astroport/oracle/src/mock_querier.rs

```diff
@@ -113,6 +136,66 @@ impl WasmMockQuerier {
                             };
                             SystemResult::Ok(to_binary(&balance).into())
                         }
+                        QueryMsg::Simulation {
+                            offer_asset,
+                            ask_asset_info: _,
+                        } => SystemResult::Ok(
+                            to_binary(&SimulationResponse {
+                                return_amount: offer_asset.amount,
+                                spread_amount: Uint128::zero(),
+                                commission_amount: Uint128::zero(),
+                            })
+                            .into(),
+                        ),
+                        _ => panic!("DO NOT ENTER HERE"),
```

Comment:
there is a standard macro for situations like this
```suggestion
                        _ => unreachable!(),
```

Reply:
I really wouldn't waste time on it atm. this code is taken as is from the origin — astroport dudes